pcs/cli/resource/parse_args.py

```diff
@@ -30,7 +30,7 @@ def parse_create_simple(arg_list):
 def parse_clone(arg_list, promotable=False):
     parts = {
         "clone_id": None,
-        "meta": {},
+        "clone_option": {},
```

Comment:
This change is unnecessary, the arguments parsed into this key are really meta attributes of a clone.